src/Uno.UWP/Helpers/AndroidResourceNameEncoder.cs

```diff
@@ -31,7 +33,7 @@ public static string Encode(string key)
 			key = new string(charArray);
 
 			//Checks if the keys are starting by a number because they are invalid in C#
-			if (int.TryParse(key.Substring(0, 1), out _))
+			if (!string.IsNullOrEmpty(key) && int.TryParse(key.Substring(0, 1), out _))
```

Comment:
`key` is never null in this code path.

```suggestion
			if (key.Length > 0 && int.TryParse(key.Substring(0, 1), out _))
```